tools/dataMigration/migrateData/_repos.ts

```diff
@@ -72,4 +72,59 @@ export const isBasicTable = (tableName: string): boolean =>
     'biomassStockTotal',
     'carbonStockAvg',
     'carbonStockTotal',
+    'country_comments_1_1_1',
```

Comment:
can this be moved to 'panEuropeanTables' and extended here like '...panEuropeanTables'

Reply:
This is just temporary